frontend/libs/health-department/feature-case-detail/src/lib/contact-list/contact-list.component.ts

```diff
@@ -120,6 +127,7 @@ export class ContactListComponent implements OnInit {
       status: listItem.caseStatusLabel,
       isHealthStaff: this.getBooleanText(listItem.isHealthStaff),
       isSenior: this.getBooleanText(listItem.isSenior),
+      remark: listItem.remark,
```

Comment:
Hast du getestet, ob auch längere Kommentare vollständig und korrekt gerendert werden in der Tabelle?

Reply:
Hab es nochmal angepasst. Längere Kommentare sind jetzt immer sichtbar.